accord-core/src/main/java/accord/messages/ReplyContext.java

```diff
@@ -0,0 +1,5 @@
+package accord.messages;
+
+public interface ReplyContext
```

Comment:
Could you link to the C* implementation part this maps to? I had sort of imagined it would be a `long` on the C* side as well, given the `RequestCallbacks` uses a long for `messageId`

Reply:
here you go: https://github.com/bdeggleston/cassandra/blob/accord-integration/src/java/org/apache/cassandra/service/accord/AccordMessageSink.java#L100

In C*, the reply context is the incoming message. After the netty refactor, the message is used to create the reply, and auto-fills stuff like the response verb, the id of course, and some timeout related stuff iirc